lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -325,6 +325,13 @@ def _section_certificates(course):
         for certificate in GeneratedCertificate.get_unique_statuses(course_key=course.id)
     }
 
+    # Get the count of all course verified users with audit passing and audit not passing statuses.
+    verified_users_with_audit_certs = len(
+        CourseEnrollment.objects.users_enrolled_in(course.id, mode='verified').filter(
```

Comment:
If this is a queryset, you should use the [`count`](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.count) method on it to get this number.